src/intervals/arithmetic.jl

```diff
@@ -153,6 +153,61 @@ function /(a::Interval{T}, b::Interval{T}) where T<:Real
     end
 end
 
+function extended_div(a::Interval{T}, b::Interval{T}) where T<:Real
```

Comment:
This function seems far too complicated -- I think it should just be able to use the `/` function that is already defined.

Reply:
Although maybe your version is slightly more efficient -- would need benchmarks to check.

Reply:
Should I make changes to the `/` function instead?